idb/dummy.go

```diff
@@ -503,6 +492,12 @@ type AlgoUpdate struct {
 	Closed bool
 }
 
+// AccountDataUpdate encodes an update or remove operation on an account_data json key.
+type AccountDataUpdate struct {
+	Update bool        // true if update, false if delete
```

Comment:
Consider changing this field to `Delete`. That way, the code won't have mixed semantics (Using the term update for deletion).

Reply:
Good idea.